### DISCUSSION TITLE
Windows Subsystem for Android (WSA) Version 2307.40000.5.0 has been released

### DISCUSSION THREAD

**Discussion by Almighty188** [LSPosed/MagiskOnWSALocal#655] [General]
Windows Subsystem for Android (WSA) Version 2307.40000.5.0 has been released

What’s New​
Platform stability improvements.
Updated latest Chromium WebView to version 115.
Android 13 security updates.

More Information:
https://blogs.windows.com/windows-insider/2023/08/17/update-to-windows-subsystem-for-android-on-windows-11-august-2023/

1) On the Windows side, download it from:
https://github.com/MustardChef/WSAPackages/releases/download/WSA_2307.40000.5.0/MicrosoftCorporationII.WindowsSubsystemForAndroid_2307.40000.5.0_neutral_._8wekyb3d8bbwe.Msixbundle
with a big thanks to @MustardChef for providing the file archive.
2) Rename the downloaded file to wsa-WIF.zip
3) Delete everything inside the download folder for MagiskOnWSALocal
4) Place the wsa-WIF.zip file in the MagiskOnWSALocal download folder
*** Note *** Build is broken per [Bug] version 2306 build failed [Bug] version 2306 build failed #615 but can be built by manually making changes to scripts/build.sh with many thanks to @YT-Advanced as seen here:
https://github.com/LSPosed/MagiskOnWSALocal/pull/620/files
5) In Linux, Build MagiskOnWSALocal as usual and remember to choose "insider fast" for WSA release type.

**Comment by simon777736** [comment]
On step 5, is mean run build.sh then run run.sh?

**Reply by MustardChef** [reply]
@simon777736 Or you can clone: https://github.com/WSABuilds/MagiskOnWSALocal.git and run ``run.sh``
Then you select "latest" for the latest insider build (2307.40000.5.0), and then choose whatever else you want.

**Reply by MustardChef** [reply]
Usage:

```shell
cd ~/
sudo git clone https://github.com/WSABuilds/MagiskOnWSALocal.git
cd MagiskOnWSALocal
cd scripts
sudo ./run.sh
```

**Reply by Almighty188** [reply]
Whichever way you desire as that is up to your own preference which is why I said build it like usual and not say to use a specific method to build but make sure for the WSA build to specify as Fast Channel.  build.sh is the file you have to modify in step 4.  Step 5 is either run.sh or build.sh obviously.

**Reply by simon777736** [reply]
thanks!

**Reply by im1kh** [reply]
[build.zip](https://github.com/LSPosed/MagiskOnWSALocal/files/12503353/build.zip)
 
this is build.sh for 2307 worked for me, after editing. 
it took awhile to fix but i hope you reach this first to save your time from erors.
